grim.go

```diff
@@ -146,6 +151,17 @@ func (i *Instance) BuildNextInGrimQueue(logger *log.Logger) error {
 	return nil
 }
 
+func repoIsAlive(hook *hookEvent) bool {
+	url := hook.URL
+	if len(url) >= 12 {
```

Comment:
I think this need to be:
return len(url) >= 12 && url[len(url)-12:] != "000000000000"